bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -493,7 +493,8 @@ export default {
               ? [this.$oh.api.get('/rest/ui/components/ui:page'), this.$oh.api.get('/rest/ui/components/ui:widget')]
               : [Promise.resolve([]), Promise.resolve([])],
             dayjsLocalePromise,
-            this.$store.dispatch('loadSemantics')
+            this.$store.dispatch('loadSemantics'),
+            this.$store.dispatch('loadSemanticModel')
```

Comment:
Okay I now understood why you decided to load the model inside app.vue - so that it is available for the model tabs outside of the home page.
I have just pushed a commit that refactors model loading, so that the model is loaded by app.vue and reloaded on Item change and the model is not loaded anywhere else.